menas/src/main/scala/za/co/absa/enceladus/menas/Application.scala

```diff
@@ -31,9 +31,11 @@ class Application() {
 
   @Bean def asyncExecutor(): ThreadPoolTaskExecutor = {
     val executor = new ThreadPoolTaskExecutor()
+    // scalastyle:off magic.number
     executor.setCorePoolSize(12)
```

Comment:
I think issues with these magic numbers should be addressed separately and properly by moving them to constants or configuration. If we just suppress them we are going to forget about it.